### ISSUE TITLE
subtly indicate to user that updates have been made

### ISSUE THREAD

**Issue by hchiam** [hchiam/html-template-generator#73]
e.g.: 1.2.3 -> sum -> mod 9 -> mapping to some colour

https://stackoverflow.com/questions/10014271/generate-random-color-distinguishable-to-humans/10014411

https://en.wikipedia.org/wiki/Golden_angle

https://stackoverflow.com/questions/470690/how-to-automatically-generate-n-distinct-colors
﻿

**Comment by hchiam** [comment]
swap lime with different generated colour?﻿

**Comment by hchiam** [comment]
i'll go with hard-coded colours to ensure WCAG AA contrast with black text
